docs/content.zh/docs/connectors/datastream/formats/parquet.md

```diff
@@ -198,14 +198,14 @@ ds = env.from_source(source, WatermarkStrategy.no_watermarks(), "file-source")
 
 Flink 支持三种方式来读取 Parquet 文件并创建 Avro records （PyFlink 只支持 Generic record）：
 
-- [Generic record](https://avro.apache.org/docs/1.10.0/api/java/index.html)
-- [Specific record](https://avro.apache.org/docs/1.10.0/api/java/index.html)
-- [Reflect record](https://avro.apache.org/docs/1.10.0/api/java/org/apache/avro/reflect/package-summary.html)
+- [Generic record](https://avro.apache.org/docs/++version++/api/java/index.html)
```

Comment:
why not point to https://avro.apache.org/docs/++version++/api/java/org/apache/avro/generic/package-summary.html - same for specific record.